app/src/main/java/org/squidfish/tuplenback/TupleNBackApplication.kt

```diff
@@ -4,15 +4,18 @@ import android.app.Application
 import org.koin.android.ext.koin.androidContext
 import org.koin.core.context.startKoin
 import org.koin.ksp.generated.module
+import org.squidfish.tuplenback.di.DatabaseModule
 import org.squidfish.tuplenback.di.RepositoryModule
 import org.squidfish.tuplenback.di.ViewModelModule
 
 class TupleNBackApplication: Application() {
     override fun onCreate() {
         super.onCreate()
+
         startKoin {
             androidContext(this@TupleNBackApplication)
             modules(
+                DatabaseModule().module,
```

Comment:
Since it's not very likely to have more than one database in this app, maybe it should be `DataSourceModule`? This is just a suggestion